azblob/url_blob.go

```diff
@@ -108,16 +120,22 @@ func (b BlobURL) Undelete(ctx context.Context) (*BlobUndeleteResponse, error) {
 // does not update the blob's ETag.
 // For detailed information about block blob level tiering see https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers.
 func (b BlobURL) SetTier(ctx context.Context, tier AccessTierType, lac LeaseAccessConditions) (*BlobSetTierResponse, error) {
-	return b.blobClient.SetTier(ctx, tier, nil, RehydratePriorityNone, nil, lac.pointers())
+	return b.blobClient.SetTier(ctx, tier, nil,
+		nil, // Blob versioning
```

Comment:
>// Blob versioning [](start = 7, length = 18)

please change these comments to clarify that the blob version is already included in the URL

Reply:
Done